iphone/Maps/UI/PlacePage/Components/WikiDescriptionViewController.swift

```diff
@@ -18,6 +18,16 @@ class WikiDescriptionViewController: UIViewController {
 
     descriptionTextView.textContainerInset = .zero
     updateDescription()
+    
+    let longPressGesture = UILongPressGestureRecognizer(target: self, action: #selector(handleLongPress))
+      descriptionTextView.addGestureRecognizer(longPressGesture)
+  }
+  
+  @objc func handleLongPress(sender: UILongPressGestureRecognizer) {
+    if sender.state == .began {
+      UIPasteboard.general.string = descriptionTextView.text
+      Toast.toast(withText: "Copied!").show()
```

Comment:
Please reuse the translated string `copied_to_clipboard` (also check its usage in the code).

Reply:
ok

Reply:
Sir do I have to find (copied_to_clipboard) in Localizable(en)
Or just  directly write copied_to_clipboard in the Toast message

Reply:
1. Find it in data/strings/strings.txt and check how is it used in Android code (the copied text should be appended to the string).
2. Use it in the code on iOS.
3. Regenerate strings with toos/unix/generate_translations.sh (see docs/TRANSLATIONS.md)
4. Test the change and the toast in light and dark mode.